### ISSUE TITLE
Home app iOS 14 not seeing some of my HS3 devices

### ISSUE THREAD

**Issue by SteveQ2** [jvmahon/Homebridge-HomeSeer4#144]
I have been using HomeSeer HS3 and HomeBridge for about 2 years with only minor issues.  I’m not sure but I think the most recent update to iOS 14 on my iPhone 6 broke the HS3/HomeBridge/Home app interface.  The Home app showed “no response” to all my HS3 devices.  So I saved my config.JSON files and completely uninstalled HomeBridge (2 instances).  After restarting the computer I reinstalled HomeBridge and my config.JSON file.  I think I installed the latest updates to HomeBridge. 

Now, some of my devices are working on the Home app but many still show “no response”.
![D7B57C8D-972C-4F44-A4C8-129C16AE5243](https://user-images.githubusercontent.com/72275254/118873828-33f32c80-b8b8-11eb-81e0-fa77e50f6b3c.jpeg)

So far, I cannot find anything in common with the devices that don’t work. 

I’ve run out of troubleshooting ideas.  Suggestions would be much appreciated. 

Steve

PS My cameras also don’t work!

**Comment by jvmahon** [comment]
Unfortunately, I've sometimes found the only fix in these situations is to delete the "Home" from your iOS device and also delete the homebridge data from your device running homebridge and set everything up again.

See section "F" of this wiki entry: https://github.com/jvmahon/Homebridge-HomeSeer4/wiki/Windows-10-Installation

Or section "D" of this one: https://github.com/jvmahon/Homebridge-HomeSeer4/wiki/Linux-Installation-and-Errors

**Comment by jvmahon** [comment]
No response so I closed this.

**Comment by SteveQ2** [comment]
Thanks for sending the links.

I was able to restore my HomeKit setup on my iPhone by deleting the “Bridge” (the one using the HomeBridge Accessories). I then added it  back via the scan code.  Although I had to put the accessories back into their respective room and set them as a switch or bulb, this was quite easy.  I did not have to do anything to HomeBridge 😀👍👍.  It was pretty quick and painless. 

Thanks for your help. 
Steve Q
